pkgs/games/freedroid/default.nix

```diff
@@ -0,0 +1,57 @@
+{ lib, stdenv
+, fetchFromGitHub
+, makeDesktopItem, copyDesktopItems
+, imagemagick
+, autoconf, automake
+, SDL, SDL_mixer, SDL_image, SDL_gfx
+, libvorbis
+, libjpeg, libpng
+, zlib
+}:
+
+stdenv.mkDerivation rec {
+  pname   = "freedroid";
+  version = "1.2.1";
+
+  src = fetchFromGitHub {
+    owner  = "ReinhardPrix";
+    repo   = "FreedroidClassic";
+    rev    = "release-${version}";
+    sha256 = "027wns25nyyc8afyhyp5a8wn13x9nlzmnqzqyyma1055xjy5imis";
+  };
+
+  nativeBuildInputs = [ autoconf automake copyDesktopItems imagemagick ];
+  buildInputs = [
+    SDL SDL_image SDL_gfx
+    (SDL_mixer.override { enableMusicMod = true; })
+    libjpeg libpng libvorbis zlib
+  ];
+
+  postPatch = "touch NEWS";
+
+  preConfigure = "./autogen.sh";
+
+  postInstall = ''
+    mkdir -p $out/share/icons/hicolor/32x32/apps
+    convert graphics/paraicon.bmp $out/share/icons/hicolor/32x32/apps/freedroid.png
+  '';
+
+  desktopItems = [ (makeDesktopItem {
+    name = "${pname}";
+    exec = "${pname}";
+    icon = "${pname}";
```

Comment:
```suggestion
    name = pname;
    exec = pname;
    icon = pname;
```

Reply:
Oops. I'll give myself an "unnecessary use of quotation and interpolation" award. :-)